src/components/PopUpWithConfirm.js

```diff
@@ -0,0 +1,27 @@
+import PopUp from "./PopUp.js";
+
+export default class PopUpWithConfirm extends PopUp {
+  constructor(popUpSelector, handleFormSubmit) {
+    super({ popUpSelector });
+    this._handleFormSubmit = handleFormSubmit;
+    this._popUpForm = this._popUpElement.querySelector(".modal__form");
+  }
+
+  open(card) {
+    this.card = card;
+    console.log(`open(card): this.card._id = ${this.card._id}`);
+    super.open();
+  }
+
+  close() {
+    super.close();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.